doc/content/getting-started/platform-guide/_index.md

```diff
@@ -5,27 +5,87 @@ weight: -1
+{{< figure src="gnse_claim_0.png" alt="Create new Application" >}}
+
+
+Once you have a test applicatin similar to this we can move to the next step:
+{{< figure src="gnse_claim_1.png" alt="Test application" >}}
+
+
+## Step 3: Claim your device ownership
+
+{{% gnse %}} comes pre-flashed from the factory which the vanilla app that uses the on-board secure element to join the LoRaWAN network using over the air activation (OTAA) method.
+
+In order to register your device and claim its ownership all you have to do is to use information that's printed on the back of each device during the end device registration process. 
+
+We click on **Register end device** button to create a new one. Here we have two options:
+{{< figure src="gnse_claim_2.png" alt="Registering new end device" >}}
+
+### Step 4: Claiming by QR code 
+
+If your PC/Laptop has a webcam then this is the easiest option. Just click on the **Scan end device QR code** button (w/ camera icon) and select your webcam input and scan the back of your device. You may also use a smartphone to accomplish this. 
+
+Once the camera is in focus, console would be able to successfuly scan the QR code and extract the required device credentials. Please hit **Apply** to continue the registration process.
+
+{{< figure src="gnse_claim_3.png" alt="QR code scanned" >}}
+
+{{% gnse %}} is already included in the [device repository](https://www.thethingsnetwork.org/device-repository/), so you just input the device brand, model, region profile and the frequency plan for your location (i.e. The Things Industries -> Generic Node (SE), EU_863_870, SF9 for RX2 - recommended ) and simply hit **Register end device**.
+
+{{< figure src="gnse_claim_4.png" alt="Registering end device" >}}
+
+Congratulations! You have successfully registered your {{% gnse %}} device. You may proceed to ***Step 5***.
+
+{{< figure src="gnse_claim_7.png" alt="Device registered" >}}
+
+
+## Step 5: Powering up your device
+
+After claiming, you can power up the device using two standard AA batteries or with a small 3.7V LiPo/Li-ion battery with a standard JST PH 2.0 connector.
+
+If the device powers up correctly, you will see the green LED briefly lighting up indicating successful boot-up. Pressing the front user button also triggers a sensor sampling followed by an uplink.
+
+## Step 6: Joining the network and seeing the first data packets
+
+The device will attempt to join LoRaWAN network using OTAA. Please make sure there's a TTN/TTS LoRaWAN Gateway coverage in your area. 
+
+When the device joins successfully you will see some verbose activity and raw payload in the ***Live Data*** tab of the device console. If you are not able to understand the raw bytes, don't panic!, it's because we still need to modify the payload formatter. Please copy-paste the custom Javascript formatter code below to the ***Payload Formatters -> Uplink*** section of your application. 
```

Comment:
```suggestion
When the device successfully joins, you will see some verbose activity and raw payload in the **Live Data** tab in your device's overview page. If you are not able to understand the raw bytes, don't panic! It's because you still need to configure the payload formatter. Please copy-paste the following JavaScript formatter code below to the **Payload formatters -> Uplink** section. You can configure a payload formatter on a device or on an application level. 
```